api/src/modules/import-data/sourcing-data/dto-processor.service.ts

```diff
@@ -174,6 +182,8 @@ export class SourcingRecordsDtoProcessorService {
       sourcingData.push(sourcingLocationWithSourcingRecords as SourcingData);
     }
 
+    await this.validateCleanData(sourcingData);
```

Comment:
This is a very personal opinion, but I think it would be more readable if we take this out of the <cleanData> process